src/d1to2fix/symbolsearch.d

```diff
@@ -63,7 +72,15 @@ public void initializeModuleCache(string[] paths)
                 || symbol.kind == CompletionKind.interfaceName
                 || symbol.kind == CompletionKind.className)
             {
-                collectNames((*symbol)[]);
+                try
+                {
+                    collectNames((*symbol)[]);
+                }
+                catch (SymbolSearchRecursionError e)
+                {
+                    e.add(symbol.name ~ "@" ~ symbol.symbolFile);
+                    throw e;
+                }
```

Comment:
That's a lot of throw / catch for nothing. For a limit of 5000, do I understand right that we will catch 5k times, but only do something meaningful 20 times ?